test/unit/puppetlabs/services/jruby_pool_manager/jruby_agents_test.clj

```diff
@@ -68,7 +74,10 @@
             jruby-instance
             pool-context
             :with-jruby-retry-test
-            (is (instance? JRubyInstance jruby-instance))))
+            (is (instance? JRubyInstance jruby-instance)
+                (format
+                 "Unable to borrow instance after retries.  Num borrows: %d"
+                 @num-borrows))))
```

Comment:
@rlinehan I think that's a great idea, thanks.  I pushed a separate commit, 03b0638, which sets the default borrow timeout in the `jruby-config` test helper to 5 minutes, hopefully shorter than a Jenkins or Travis timeout but plenty long enough for what the various tests in this suite need to do.